index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portofolio</title>
+</head>
+<body>
+
+  <div class="main-content">
+  <h1>Hey There.</h1>
+  <h1>I'm Wycliff</h1>
+  <h2>I'm software developer!</h2>
```

Comment:
- [ ] Kindly encase this paragraph in a `<p>` tag as it's much more semantically correct.